app/models/concerns/turbo/broadcastable.rb

```diff
@@ -67,15 +67,69 @@
 #       end
 #   end
 #
-# There are four basic actions you can broadcast: <tt>remove</tt>, <tt>replace</tt>, <tt>append</tt>, and
-# <tt>prepend</tt>. As a rule, you should use the <tt>_later</tt> versions of everything except for remove when broadcasting
+# There are seven basic actions you can broadcast: <tt>after</tt>, <tt>append</tt>, <tt>before</tt>,
+# <tt>prepend</tt>, <tt>remove</tt>, <tt>replace</tt>, and
+# <tt>update</tt>. As a rule, you should use the <tt>_later</tt> versions of everything except for remove when broadcasting
 # within a real-time path, like a controller or model, since all those updates require a rendering step, which can slow down
 # execution. You don't need to do this for remove, since only the dom id for the model is used.
 #
-# In addition to the four basic actions, you can also use <tt>broadcast_render</tt>,
+# In addition to the seven basic actions, you can also use <tt>broadcast_render</tt>,
 # <tt>broadcast_render_to</tt> <tt>broadcast_render_later</tt>, and <tt>broadcast_render_later_to</tt>
 # to render a turbo stream template with multiple actions.
 #
+# == Page refreshes
+#
+# You can broadcast a "page refresh" to all clients subscribed to a stream. This is useful when you need to
+# broadcast a lot of partial updates, and the entire content is not easily updatable using Turbo stream actions like replace, append, prepend, or remove.
+#
+# It's recommended to use page refreshes as the primary option for broadcasting changes.
+# This approach allows you to remove the coupling between models and views to render complex partial updates without sacrificing the desired high UI fidelity.
+# We hope this makes using the other available Turbo Stream actions less necessary.
```

Comment:
```suggestion
# You can broadcast "page refresh" stream actions. This will make subscribed clients reload the 
# page. For pages that configure morphing and scroll preservation, this will translate into smooth
# updates when it only updates the content that changed.

# This approach is an alternative to fine-grained stream actions targeting specific DOM elements. It
# offers good fidelity with a much simpler programming model. As a tradeoff, the fidelity you can reach
# is often not as high as with targeted stream actions since it renders the entire page again.
```